controllers/tutorialsControllers.js

```diff
@@ -7,48 +7,52 @@ const tutorialsAll = async (req, res) => {
       createdAt: -1,
     });
     res.json(tutorials);
+    res.status(200).send();
   } catch (error) {
-    res.json({ message: error });
+    res.status(404).send();
   }
 };
 
 // Add new Tutorials
 const tutorialsCreate = async (req, res) => {
   const tutorial = new Tutorial({
+    title: req.body.title,
+    description: req.body.description,
     published: req.body.published,
   });
   try {
     const savedTutorial = await tutorial.save();
     res.send(savedTutorial);
   } catch (error) {
-    res.status(400).send(error);
+    res.status(405).send(error);
   }
 };
 
 // Update a Tutorial by ID
 const updateTutorials = async (req, res) => {
   try {
-    const { update } = req.body;
-    const { options } = { new: true };
+    const { title, description, published } = req.body;
+    const update = {};
+    if (typeof title !== "undefined") update.title = title;
+    if (typeof description !== "undefined") update.description = description;
+    if (typeof published !== "undefined") update.published = published;
     const tutorialUpdate = await Tutorial.findByIdAndUpdate(
-      { _id: req.params.tutorialId },
-      { published: req.body.published },
+      { _id: req.params.Id },
       update,
-      options
+      { new: true }
     );
     res.json(tutorialUpdate);
```

Comment:
Don't need to return anything here. Just Response with status code 203 should befine.
```
POST /tutorials - Should return 201 with newly created record.
PATCH /tutorials/:id - Should return 201 without any response.
DELETE /tutorials/:id - Should return 200 without any response.
```
CC: @ashishkava @JainishaParmar